controller.go

```diff
@@ -395,5 +441,16 @@ func (c *controller) isDriverGlobalScoped(networkType string) (bool, error) {
 }
 
 func (c *controller) GC() {
-	sandbox.GC()
+	platform.GC()
+}
+
+// GetSbxWlkContainerID returns a Sandbox Walker function which looks for an existing Sandbox with the passed containerID
```

Comment:
Being an external facing API, I think it is okay to expand the function name. `SandboxContainerWalker`